test/e2e_test_attach.sh

```diff
@@ -59,24 +67,22 @@ echo "Payload: $PAYLOAD"
 
 ## Attach Signature, payload, cert and cert-chain
 ./cosign attach signature --signature ./payloadbase64.sig --payload ./payload.json --cert ./leafcert.pem --cert-chain ./certchain.pem $IMAGE_URI_DIGEST
+./cosign attach signature --signature ./secondpayloadbase64.sig --payload ./payload.json --cert ./secondleafcert.pem --cert-chain ./secondcertchain.pem $IMAGE_URI_DIGEST
```

Comment:
Can we modify this test so that we still test verification with a single attached cert, then after verification, attach another and try verification?